examples/quantization/brevitas/quantize_llm.py

```diff
@@ -1,13 +1,9 @@
 from argparse import ArgumentParser
 
-import torch
-from brevitas.export.onnx.standard.qcdq.manager import StdQCDQONNXManager
-from brevitas_examples.llm.llm_quant.export import brevitas_proxy_export_mode
-
 from optimum.amd import BrevitasQuantizationConfig, BrevitasQuantizer
 from optimum.amd.brevitas.accelerate_utils import calc_cpu_device_map, calc_gpu_device_map, offload_model, remove_hooks
 from optimum.amd.brevitas.data_utils import compute_perplexity, get_dataset_for_model
-from optimum.exporters.onnx import onnx_export_from_model
+from optimum.amd.brevitas.export import export_quantized_model
```

Comment:
```suggestion
from optimum.amd.brevitas.export import export_to_onnx
```
Can we keep the ONNX word in the loop to make it explicit. Other name suggestions, `quantized_model_to_onnx` or `save_quantized_model_as_onnx`

Reply:
I opted to keep as similar as possible to the original name so it became:
`onnx_export_from_quantized_model`